Facepunch.Steamworks/SteamRemoteStorage.cs

```diff
@@ -79,6 +79,22 @@ public unsafe static byte[] FileRead( string filename )
 		public static bool FileDelete( string filename ) => Internal.FileDelete( filename );
 
 
+		/// <summary>
+		/// Marks the file as shared so other users can download it
+		/// </summary>
+		public static async Task<ulong> FileShare(string filename)
```

Comment:
What is ulong here?

Reply:
The UgcHandle of the file that was shared